setup.cfg

```
@@ -52,7 +52,7 @@ install_requires =
     psutil>=5.0
     PyOpenGL>=3.1.0
     PyYAML>=5.1
-    qtconsole>=4.5.1
+    qtconsole>=4.7.7
```

@VolkerH could this be resolved by blocking a specific version?

```suggestion
    qtconsole>=4.5.1,!=4.7.6
```

---

probably... are there any benefits to enabling old qtconsole versions though? Given that napari has already dropped python 3.6 I don't really see a benefit.

---

But if you think otherwise, I'm happy to commit your suggestion.

---

I just checked `4.5.1`, `4.7.5` and `4.7.7` all work, but `4.7.6` does not. Given that the bug seems localized to just that version I'd follow the recommendation of @jni here and just block that specific version as it won't force users to upgrade unnecessarily

---

Thanks @jni, @sofroniewn 
Convinced. Change applied.